shared/src/Microsoft.Azure.Devices.Shared.csproj

```diff
@@ -18,7 +18,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <Version>1.27.2</Version>
+    <Version>1.31.0</Version>
```

Comment:
Agreed, this should be 1.27.3 because of how we handle versioning with LTS.

Reply:
yup, I will update to the next patch version which is 1.27.2 since this has been already updated in the previous pr